blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_de.properties

```diff
@@ -0,0 +1,64 @@
+Activity=Aktivit\u00E4ten
+Artifacts=Artefakte
+Author=Autor
+Branch=Ableger
```

Comment:
not sure how serious https://github.com/danielauener/git-auf-deutsch is but he is using Zweig instead. ```Ableger``` is more Fork I guess.